gui/default/untrusted/syncthing/core/syncthingController.js

```diff
@@ -1828,12 +1830,31 @@ angular.module('syncthing.core')
             $scope.currentFolder.path = pathJoin($scope.config.defaults.folder.path, newvalue);
         });
 
-        $scope.fsWatcherToggled = function () {
+        $scope.setFSWatcherIntervalSDefault = function () {
```

Comment:
The function name differs between untrusted and default.  The "S" needs to go here.